Awful.apk/src/main/res/layout/test_fragment.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Did you mean to commit this?

Reply:
Heh like I need to make a test fragment, heh heh *tugs collar*